VK_App/VK_App/Controller/UserGroupTableViewController.swift

```diff
@@ -16,6 +16,7 @@ final class UserGroupTableViewController: UITableViewController {
 
     // MARK: - Private properties
 
+    private var photoCacheService: PhotoCacheService?
```

Comment:
```suggestion
    lazy private var photoCacheService: PhotoCacheService(container: self)
```
Вот так можно. Ну это как вариант